flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -86,6 +86,13 @@ under the License.
 
 		<!-- test dependencies -->
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
```

Comment:
I don't understand why this dependency is necessary. It should be on the classpath in any case because the connectors have a dependency on it. This should also apply to other modules.